ddp-workspace/projects/ddp-sdk/src/lib/directives/lazyLoadResources.directive.spec.ts

```diff
@@ -0,0 +1,74 @@
+/* tslint:disable:max-classes-per-file */
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { Component, DebugElement } from '@angular/core';
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { By } from '@angular/platform-browser';
+import { LazyLoadResourcesDirective, WindowRef } from 'ddp-sdk';
+import { FormsModule } from '@angular/forms';
+
+fdescribe('LazyLoadResourcesDirective', () => {
+  @Component({
+    template: `
+        <img lazy-resource [src]="src" alt=""/>`
+  })
+  class TestHostComponent {
+    src = 'http://test';
+  }
+
+  let component: TestHostComponent;
+  let fixture: ComponentFixture<TestHostComponent>;
+  let debugElement: DebugElement;
+
+  beforeEach(waitForAsync(() => {
+    class IntersectionObserver {
+      observe: () => void;
+      unobserve: () => void;
+
+      constructor(
+        public callback: (entries: Array<IntersectionObserverEntry>) => void
+      ) {
+        this.observe = () => {
+          this.callback([{isIntersecting: true} as IntersectionObserverEntry]);
+        };
+        this.unobserve = jasmine.createSpy('unobserve');
+      }
+    }
+    /* eslint-disable  @typescript-eslint/no-explicit-any */
+    (window as any).IntersectionObserver = IntersectionObserver;
+
+    TestBed.configureTestingModule({
+      imports: [
```

Comment:
Seems like these imports are not needed.

Reply:
fixed